vmdb/app/helpers/container_project_helper/textual_summary.rb

```diff
@@ -40,4 +40,48 @@ def textual_ems
     end
     h
   end
+
+  def textual_container_routes
+    count_of_routes = @record.number_of(:container_routes)
+    label = ui_lookup(:tables => "container_routes")
```

Comment:
@kbrock if this is the only issue I'll take care of that with an additional PR after you merged this (as Mooli won't be available until Sunday).